README.md

```diff
@@ -61,16 +61,16 @@ NOTE:
 
 | Plugin | Features                                                  | Feature Status | CSI Driver Version | CSI Spec Version | Ceph Cluster Version | Kubernetes Version |
 | ------ | --------------------------------------------------------- | -------------- | ------------------ | ---------------- | -------------------- | ------------------ |
-| RBD    | Dynamically provision, de-provision Block mode RWO volume | GA             | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
```

Comment:
I think the documentation can stay at v1.13.0. But there should be a mentioning of the minimal Kubernetes version a little above this section.

Reply:
we have removed all the 1.13 templates so we dont need to mention about 1.13 anymore, and we cant use 1.14+ templates with kube 1.13

Reply:
Can do but when do we decide to update the minimal Kubernetes version in the table? Since the 1.13 e2e test are removed there is no indication a 1.13 breaking change is introduced.

Would something like this suffice?
```markdown
NOTE:

- **`csiv0.3`** is deprecated with release of **`csi v1.1.0`**
- The helm charts **require** Kubernetes 1.14+
- The minimum tested Kubernetes version is 1.14 since release v2.0.0
```

Reply:
Keeping the older versions of when a feature was introduced seems useful to me. It serves as a history and confirmation that the functionality is tested and likely deployed in production since ... version.

Reply:
if we mention that we support kube 1.13 as we are removing the templates, what if the user wants to deploy ceph-csi v2.0.0 on kube 1.13?

Reply:
I my humble opinion it is good to remove the 1.13 support since there have been some issues in the past due the sheer number of files to be edited for a change. If I need to edit the provisioner deployment in the current state I need to change:

* deploy/cephfs/1.13/csi-cephfsplugin-provisioner.yaml
* deploy/cephfs/1.14+/csi-cephfsplugin-provisioner.yaml
* deploy/rbd/1.13/csi-rbdplugin-provisioner.yaml
* deploy/rbd/1.14+/csi-rbdplugin-provisioner.yaml
* charts/ceph-csi-cephfs/provisioner-deployments.yaml
* charts/ceph-csi-cephfs/provisioner-statefulset.yaml

When a change is needed in the provisioner and the nodeplugin the files to edit is doubled. A little oversight and one of the files is forgotten.

What are the advantages of supporting a scenario the upstream wouldn't support?

Reply:
I think I read the table a little different. "Kubernetes Version" means the version when this feature was introduced, not necessarily still supported. Mentioning in the "Support Matrix" paragraph that kubernetes 1.14 or newer is required since ceph-csi 2.0.0 has my preference.

I am not strongly attached to this though, so would not strongly object to updating >=v1.13 to >=1.14 either.